model/shift_design_variables.py

```diff
@@ -2,14 +2,17 @@
 
 
 class ShiftDesignVariables:
-    def __init__(self, model, shifts, time_periods):
+    def __init__(self, model, shifts, low_dur_shifts, long_dur_shifts, time_periods):
```

Comment:
Koden blir mer clean om du får pakket `low_dur_shifts` og `long_dur_shifts` in i `shifts`-parameteren. 

Pirk, men er det ikke bedre med `short_` i stedet for `low_`?

Reply:
Enig i begge disse.